test/lit/parse-nominal-types-no-sig-sharing.wast

```diff
@@ -0,0 +1,39 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
+
+;; RUN: foreach %s %t wasm-opt --nominal -all -S -o - | filecheck %s
+;; RUN: foreach %s %t wasm-opt --nominal -all --roundtrip -S -o - | filecheck %s
```

Comment:
Do these need --closed-world?

Reply:
Ah, with closed world we'd also get a validation check here, good idea. Adding.